tests/amqp_management_ut/amqp_management_ut.c

```diff
@@ -270,6 +270,38 @@ static ASYNC_OPERATION_HANDLE my_messagesender_send_async(MESSAGE_SENDER_HANDLE
     return test_send_operation;
 }
 
+static void ASYNC_OPERATION_HANDLE_ToString(char* string, size_t bufferSize, ASYNC_OPERATION_HANDLE val)
+{
+    (void)bufferSize;
+    (void)sprintf(string, "%p", val);
+}
+
+static int ASYNC_OPERATION_HANDLE_Compare(ASYNC_OPERATION_HANDLE left, ASYNC_OPERATION_HANDLE right)
+{
+    return left != right;
+}
+
+typedef struct ASYNC_OPERATION_CONTEXT_STRUCT_TEST_TAG
+{
+    ASYNC_OPERATION_CANCEL_HANDLER_FUNC async_operation_cancel_handler;
+    unsigned char context[64]; // 64 is the size of OPERATION_MESSAGE_INSTANCE.
```

Comment:
nit: `#define`?

Reply:
Nah, it should be ok just with the comment.